src/caf/space/inputs.py

```diff
@@ -157,6 +161,59 @@ def _valid_data_col(cls, v, values):
         return v
 
 
+@dataclasses.dataclass
+class SpaceArguments:
+    """Command Line arguments for running space."""
+
+    config_path: Path
+    mode: str
+    out_path: Path
+
+    @classmethod
+    def parse(cls) -> SpaceArguments:
+        """Parse command line argument."""
+        parser = argparse.ArgumentParser(
+            description=__doc__, formatter_class=argparse.ArgumentDefaultsHelpFormatter
+        )
+        parser.add_argument(
+            "--mode",
+            type=str,
+            help="Mode to run translation in; spatial, weighted or GUI.",
+            default="GUI",
+            required=False,
```

Comment:
`add_argument` has a choices parameters which will make sure the input is one of a limited set of options e.g. `choices=MODES` will check the given value is in `MODES`.